docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+st2:
+  # st2api (9101),
+  # st2garbagecollector,
+  # st2stream (9102),
+  # st2actionrunner,
+  # st2auth (9100), 
+  # st2notifier,
+  # st2resultstracker,
+  # st2rulesengine,
+  # st2web (80)
+  image: st2:latest
```

Comment:
Talking about naming, - `st2` might be confusing for Docker image running both st2 and st2web inside with init system.

How about `st2-upstart`?
This will follow the best practices in naming Docker containers and so is more obvious for users to understand what's inside, see: https://hub.docker.com/search/?isAutomated=0&isOfficial=0&page=1&pullCount=0&q=upstart&starCount=0